### DISCUSSION TITLE
share enum types?

### DISCUSSION THREAD

**Discussion by maxandersen** [quarkiverse/quarkus-openapi-generator#544] [Q&A]
how can one go about making enums the same types?

I end up with a bunch of SomeEntity.ClearedEnum, AnotherEntity.ClearedEnum, etc. instead of just one ClearedEnum.

How does one align those?

**Comment by ricardozanini** [comment]
Good question, @maxandersen. We probably need to process that info during parsing time and generate one single instance accordingly, in the parent namespace. Mind opening an issue?

**Reply by maxandersen** [reply]
https://github.com/quarkiverse/quarkus-openapi-generator/issues/551
